sdk/relay/Azure.ResourceManager.Relay/tests/Helpers/RelayTestBase.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.ResourceManager.Relay.Models;
+using Azure.Core;
+using System.Security.Cryptography;
+using Azure.ResourceManager.Network;
+using Azure.ResourceManager.ManagedServiceIdentities;
+using Azure.ResourceManager.Relay.Tests;
+using Azure.ResourceManager.Relay;
+
+namespace Azure.ResourceManager.Relay.Tests.Helpers
+{
+    [ClientTestFixture]
```

Comment:
I assume this attribute is needed when we want to enable the multi-version testing. Any reason that we use it here? If it's not required, I'd like to remove it.